autogpt_platform/backend/backend/blocks/google/gmail.py

```diff
@@ -1396,8 +1516,10 @@ class Input(BlockSchemaInput):
             default=None,
             advanced=True,
         )
-        attachments: list[MediaFileType] = SchemaField(
-            description="Files to attach", default_factory=list, advanced=True
+        attachments: list[GoogleDriveFile] = GoogleDriveAttachmentField(
```

Comment:
why are we doing this? swapping attachments from google drive to media file is a choice but maybe we should have both